app/src/main/java/com/kickstarter/viewmodels/outputs/DiscoveryViewModelOutputs.java

```diff
@@ -30,11 +31,17 @@
    * Emits when the params of a particular page should be updated. The page will be responsible for
    * taking those params and creating paginating projects from it.
    */
-  Observable<Pair<DiscoveryParams, Integer>> updateParamsForPage();
```

Comment:
we no longer need to pass the `Integer` position here because with cleaner params, the `DiscoveryParams` contains the actual `sort` that can be used to derive the position!